explorer/deploy/webAppCreator.ts

```diff
@@ -9,9 +9,11 @@ import { ResourceManagementClient, ResourceModels, SubscriptionModels } from 'az
 import { Subscription } from 'azure-arm-resource/lib/subscription/models';
 import WebSiteManagementClient = require('azure-arm-website');
 import * as WebSiteModels from 'azure-arm-website/lib/models';
+import * as fs from 'fs';
+import * as path from 'path';
 import * as vscode from 'vscode';
 import { reporter } from '../../telemetry/telemetry';
-import { AzureImageNode } from '../models/azureRegistryNodes';
+import { AzureImageNode } from '../models/AzureRegistryNodes';
```

Comment:
>A [](start = 42, length = 1)

I see you updated to uppercase `A`, but the file name is still lowercase `a`. Can I know why you did the change?

Reply:
Seems like a typo (the class name is capital). Changed it to lowercase